BangazonAPI/BangazonAPI/Models/Employee.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Employee
+    {
+        public string FirstName { get; set; }
+
+        public string LastName { get; set; }
+
+        public int DepartmentId { get; set; }
+
+        public bool IsSuperViser { get; set; }
+
+        public Department CurrentDepartment { get; set; }
```

Comment:
From ticket 6: 
`A representation of the computer that the employee is currently using should be included in the employee representation`

Let's add a property of type `Computer` on here